crates/hc_bundle/README.md

```diff
@@ -6,42 +6,48 @@
 
 [![License: Apache-2.0](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)
 
-Utilities to create DNA and hApp bundle files from a source working directory, and vice-versa.
-This crate defines two separate subcommands for the `hc` CLI tool, one for each type of bundle.
-Both subcommands are very similar and have identical interfaces.
+Utilities to create DNA, hApp, and web-hApp bundle files from a source working directory and manifest file, and vice-versa.
+This crate defines three separate subcommands for the `hc` CLI tool, one for each type of bundle.
+All subcommands are very similar and have nearly identical flags and options.
 
-This crate also defines standalone binaries for each subcommand, `hc-dna` and `hc-app`.
+This crate also defines standalone binaries for each subcommand, `hc-dna`, `hc-app`, and `hc-web-app`.
 
 Usage instructions from the `-h` flag:
 
 ```sh
 $ hc dna -h
-
-hc-dna 0.0.1
-Work with DNA bundles
+holochain_cli_bundle 0.1.3
```

Comment:
ditto this.

Reply:
You could put `holochain_cli_bundle <VERSION>` to prevent frequent outdating.